frontend/package.json

```diff
@@ -84,9 +85,13 @@
         "test": "react-scripts test",
         "eject": "react-scripts eject",
         "test:precommit": "npm test -- --watchAll=false",
-        "cy": "cypress open",
-        "cy:run": "cypress run",
-        "cy:precommit": "start-server-and-test start http://localhost:3000 cy:run",
+        "cy:desktop": "cypress open",
+        "cy:mobile": "cypress open --config viewportWidth=375,viewportHeight=667",
+        "cy:run:desktop": "cypress run",
+        "cy:run:mobile": "cypress run --config viewportWidth=375,viewportHeight=667",
+        "cy:precommit": "npm run cy:precommit:desktop && npm run cy:precommit:mobile",
+        "cy:precommit:desktop": "start-server-and-test start http://localhost:3000 cy:run:desktop",
+        "cy:precommit:mobile": "start-server-and-test start http://localhost:3000 cy:run:mobile",
```

Comment:
Sadly I didn't find any solution to start the server once and test both desktop & mobile. We'll have to start it twice